test/minter-suite-minters/MinterMerkle/MinterMerkleV2.test.ts

```diff
@@ -165,6 +174,11 @@ describe("MinterMerkleV1", async function () {
     // mock ERC20 genArt721Core
     const ERC20Factory = await ethers.getContractFactory("ERC20Mock");
     this.ERC20Mock = await ERC20Factory.deploy(ethers.utils.parseEther("100"));
+
+    fakeDelegationRegistry = await smock.fake("IDelegationRegistry", {
+      address: "0x00000000000076A84feF008CDAbe6409d2FE638B",
+    });
+    fakeDelegationRegistry.checkDelegateForContract.returns(true);
```

Comment:
@mchrupcala rather than adding this as a shared before-each assumption, can we please just set this assumption on a per-test level to make things more explicit / more in alignment with assumptions of each individual test? I think this will improve read-ability.

Reply:
Agreed! And this way the next time somebody writes a delegation test they won't inherit the `.returns(true)`. One more note- I added a comment to mention the deployed address we're mocking for the delegation contract.

Would you like me to go ahead and merge this? I'll request another round of review from Ryley in the meantime for when he's back.

Reply:
Fantastic–yep this LGTM to merge. 🙏